client/Components/Decks/DeckSummary.jsx

```diff
@@ -132,37 +132,48 @@ class DeckSummary extends React.Component {
-                        <div className='info-row row'><span><Trans>Actions</Trans>:</span><span className='pull-right'>{ cardCounts.action } <Trans>cards</Trans></span></div>
-                        <div className='info-row row'><span><Trans>Artifacts</Trans>:</span><span className='pull-right'>{ cardCounts.artifact } <Trans>cards</Trans></span></div>
-                        <div className='info-row row'><span><Trans>Creatures</Trans>:</span><span className='pull-right'>{ cardCounts.creature } <Trans>cards</Trans></span></div>
-                        <div className='info-row row'><span><Trans>Upgrades</Trans>:</span><span className='pull-right'>{ cardCounts.upgrade } <Trans>cards</Trans></span></div>
+                        <div className='info-row row'><span><Trans>Actions</Trans>:</span><span
+                            className='pull-right'>{ cardCounts.action } <Trans>cards</Trans></span></div>
+                        <div className='info-row row'><span><Trans>Artifacts</Trans>:</span><span
+                            className='pull-right'>{ cardCounts.artifact } <Trans>cards</Trans></span></div>
+                        <div className='info-row row'><span><Trans>Creatures</Trans>:</span><span
+                            className='pull-right'>{ cardCounts.creature } <Trans>cards</Trans></span></div>
+                        <div className='info-row row'><span><Trans>Upgrades</Trans>:</span><span
+                            className='pull-right'>{ cardCounts.upgrade } <Trans>cards</Trans></span></div>
                         <div className='info-row row'><span><Trans>Validity</Trans>:</span>
-                            <DeckStatus className='pull-right' status={ this.props.deck.status } />
+                            <DeckStatus className='pull-right' status={ this.props.deck.status }/>
                         </div>
                         { this.props.deck.usageLevel > 0 && !this.props.deck.verified ?
-                            <div className='info-row row'><Trans i18nKey='decksummary.toverify'>This deck has been flagged as requiring verification. Please email a photo of the decklist with your username written on a piece of paper to</Trans>: thecrucible.deckcheck@gmail.com
+                            <div className='info-row row'><Trans i18nKey='decksummary.toverify'>This deck has been
+                                flagged as requiring verification. Please email a photo of the decklist with your
+                                username written on a piece of paper to</Trans>: thecrucible.deckcheck@gmail.com
                             </div> : null
                         }
                     </div>
-                    <div className='col-xs-2 col-sm-3 no-x-padding'>{ this.props.deck.agenda && this.props.deck.agenda.code ? <img className='img-responsive' src={ '/img/cards/' + this.props.deck.agenda.code + '.png' } /> : null }</div>
+                    <div
```

Comment:
a better way to break this would be to keep the div attributes on one line and then newline before the {